README.md

```diff
@@ -22,7 +22,7 @@ Container images for the Dynatrace distribution of the OpenTelemetry Collector a
 
 - [GitHub Container Registry (GHCR)](https://github.com/Dynatrace/dynatrace-otel-collector/pkgs/container/dynatrace-otel-collector%2Fdynatrace-otel-collector)
 - [Amazon Elastic Container Registry (Amazon ECR)](https://gallery.ecr.aws/dynatrace/dynatrace-otel-collector)
-- [DockerHub Container Registry](https://hub.docker.com/repository/docker/dynatrace/dynatrace-otel-collector)
+- [Docker Hub Container Registry](https://hub.docker.com/r/dynatrace/dynatrace-otel-collector)
```

Comment:
Note that this link is expected to 404 until our next release where we publish it there the first time.

Reply:
excluded in 905d1f2